SourceKitStressTester/Sources/Common/Message.swift

```diff
@@ -496,9 +502,9 @@ extension SourceKitError: CustomStringConvertible {
         \(markSourceLocation(of: request) ?? "<unmodified>")
         -- end file content ----------
         """
-    case .softTimeout(let request, let duration):
+    case .softTimeout(let request, let duration, let instructions):
       return """
-        Request took \(duration) seconds to execute. This is more than half of the allowed time. This error will match XFails but won't count as an error by itself.
+        Request took \(duration) seconds (\(instructions.map(String.init) ?? "<unknown>") instructions) to execute. This is more than half of the allowed time (in seconds). This error will match XFails but won't count as an error by itself.
```

Comment:
The additional "(in seconds)" is a little confusing to me here since we don't actually say what the timeout was.